integration/spark/src/main/scala/org/apache/spark/sql/execution/command/index/DropIndexCommand.scala

```diff
@@ -184,10 +184,12 @@ private[sql] case class DropIndexCommand(
         parentCarbonTable = getRefreshedParentTable(sparkSession, dbName)
         val indexMetadata = parentCarbonTable.getIndexMetadata
         if (null != indexMetadata && null != indexMetadata.getIndexesMap) {
-          val hasCgFgIndexes =
-            !(indexMetadata.getIndexesMap.size() == 1 &&
-              indexMetadata.getIndexesMap.containsKey(IndexType.SI.getIndexProviderName))
-          if (hasCgFgIndexes) {
+          // check if any CG or FG index exists. If not exists,
```

Comment:
For example, create 2 bloom indexes, Drop both of them. In last index drop, `indexMetadata` will be null at this point(line 186). We do not seem to set `indexExists` property to `false` in that case.

Reply:
ok, handle the scenario when no cg or fg index exists then set `indexExists` property as false. Earlier this case was not handled when we drop all indexes then we didn't set the `indexExists` property as false.